src/pages/home/report/ContextMenu/BaseReportActionContextMenu.tsx

```diff
@@ -358,35 +349,6 @@ function BaseReportActionContextMenu({
     );
 }
 
-export default withOnyx<BaseReportActionContextMenuProps, BaseReportActionContextMenuOnyxProps>({
-    betas: {
-        key: ONYXKEYS.BETAS,
-    },
-    reportActions: {
-        key: ({originalReportID}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${originalReportID}`,
-        canEvict: false,
-    },
-    transaction: {
-        key: ({reportActions, reportActionID}) => {
-            const reportAction = reportActions?.[reportActionID];
-            return `${ONYXKEYS.COLLECTION.TRANSACTION}${(reportAction && ReportActionsUtils.getLinkedTransactionID(reportAction)) ?? -1}`;
-        },
-    },
-})(
-    memo(BaseReportActionContextMenu, (prevProps, nextProps) => {
```

Comment:
The memo function is used to prevent unnecessary re-renders. I think we shouldn't remove it. let's apply another way to keep our performance of the App